recipes/r-qgg/meta.yaml

```diff
@@ -0,0 +1,138 @@
+{% set version = '1.1.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-qgg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qgg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qgg/qgg_{{ version }}.tar.gz
+  sha256: dff06a1f4684622a4e60f617758254d25210f5081d814bc143906cea8b297a19
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+  ignore_run_exports:
+    - r-data.table
+    - r-corpcor
+    - r-rcpparmadillo
+    - r-matrix
+    - r-rcpp
+    - r-statmod
+    - r-mass
+    - r-mcmcpack
+    - r-coda
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - {{ stdlib("c") }}
+
+  host:
+    - r-base
+    - r-mass
+    - r-mcmcpack
+    - r-matrix
+    - r-rcpp >=1.0.5
+    - r-rcpparmadillo
+    - r-coda
+    - r-corpcor
+    - r-data.table
+    - r-statmod
+    - {{ stdlib('c') }}
+    - libopenblas 
+    - libgfortran
+    
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
```

Comment:
```suggestion
```
This no longer needed - `conda skeleton cran` is a little outdated now but deleting this should allow your Windows build to progress, hopefully fixing it.